deployments/dev/config/common.yaml

```diff
@@ -68,44 +68,47 @@ pangeo:
+            'pangeo_welcome': """Welcome to hub.pangeo.io. This hub lives in Google Cloud region <code>us-central1-b</code>. It is maintained by the <a href="http://pangeo.io">Pangeo project</a> and supported by a grant from the National Science Foundation (NSF award 1740648), which includes a direct award of cloud credits from Google Cloud. The hub's configuration is stored in the github repository <a href="https://github.com/pangeo-data/pangeo-cloud-federation/">https://github.com/pangeo-data/pangeo-cloud-federation/</a>. To provide feedback and report any technical problems, please use the <a href="https://github.com/pangeo-data/pangeo-cloud-federation//issues">github issue tracker</a>.""",
+            'announcement': 'hub.pangeo.io is being decommissioned on Monday, June 22, 2020. Please see <a href="https://discourse.pangeo.io/t/migration-of-ocean-pangeo-io-user-accounts/644">Discourse</a> for more information.'
           }
 
       extraVolumes:
         - name: custom-templates
           gitRepo:
             repository: "https://github.com/pangeo-data/pangeo-custom-jupyterhub-templates.git"
-            revision: "0c799bb2f880c8b516dcf90e2a90328078d60c25"
+            revision: "39962614b3df4ca86efb29712f7e67c51308d25b"
       extraVolumeMounts:
         - mountPath: /usr/local/share/jupyterhub/custom_templates
           name: custom-templates
           subPath: "pangeo-custom-jupyterhub-templates/templates"
         - mountPath: /usr/local/share/jupyterhub/static/extra-assets
           name: custom-templates
           subPath: "pangeo-custom-jupyterhub-templates/extra-assets"
+
+      extraEnv:
+        OAUTH2_AUTHORIZE_URL: "https://pangeo.auth0.com/authorize"
+        OAUTH2_TOKEN_URL: "https://pangeo.auth0.com/oauth/token"
+        OAUTH_CALLBACK_URL: "https://staging.hub.pangeo.io/hub/oauth_callback"
+
     auth:
-      github:
-        orgWhitelist:
-          - pangeo-data
-          - pydata
-          - dask
-          - xarray-dev
-          - jupyterhub
-          - informatics-lab
-      scopes:
-        - "read:org"
-        - "read:user"
+      type: custom
+      custom:
+        className: oauthenticator.generic.GenericOAuthenticator
+        config:
+          login_service: "auth0"
+          token_url: https://pangeo.auth0.com/oauth/token
+          userdata_url: https://pangeo.auth0.com/oauth/userinfo
+          userdata_method: GET
+          username_key: nickname
+          scope:
+            - openid
+            - profile
+            - email
```

Comment:
Happy to help, but I'm not sure exactly what you're asking me to do.

Reply:
I don't think this will work on prod since the callback is to `staging`. So we either need to a) roll this change back or b) split out the config to work for both staging/prod.

Reply:
Will push up a PR to split the config. Hopefully that's all that's needed?

Reply:
That's in. Hopefully it works.

Reply:
Let me do the same for AWS now before we merge this